### PR TITLE
ReadFile improvement and ShowIncidentIndicators bugfix for 8.0+

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5714
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5715

## Description
- Improved the ReadFile script to also print the contents of the file and not just load it into context.
- Fixed the ShowIncidentIndicators script on XSOAR 8.0+. [A bug in the Threat Intel search causes it to behave differently.](https://jira-hq.paloaltonetworks.local/browse/CRTX-76281)

## Does it break backward compatibility?
No

### CODE REVIEW

File: Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get_indicators_from_incident():
     """
     incident_id = demisto.incident()['id']
     indicators_query = {
-        "incident": incident_id
+        "investigationIDs": incident_id
```
Comment (dorschw) [inline Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py:35]: https://panw-global.slack.com/archives/C011BDBV3UP/p1676973323987969?thread_ts=1676913079.424499&cid=C011BDBV3UP

**Review by dorschw — CHANGES_REQUESTED** [review]

File: Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py (hunk 1 above)
Comment (idovandijk) [reply Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py:35]: I don't have the capacity for a test playbook at the moment, this is a quick bugfix for something seen on XSOAR 8

Comment (idovandijk) [reply Packs/CommonScripts/Scripts/ShowIncidentIndicators/ShowIncidentIndicators.py:35]: Also this is a field display script so I don't think there's a good way to test it in a test playbook, however I tested it both on 6.10 and on 8.2

File: Packs/CommonScripts/ReleaseNotes/1_11_32.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### ReadFile
+- Improved the script to also print the data of the read file to the war-room.
```
Comment (dorschw) [inline Packs/CommonScripts/ReleaseNotes/1_11_32.md:5]:
```suggestion
- Updated the script to print the file content to the war-room.
```

**Review by dorschw — APPROVED** [review]

File: Packs/CommonScripts/ReleaseNotes/1_11_32.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+
+##### ReadFile
+- Updated the script to print the file content to the war-room.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_11_32.md:5]:
```suggestion
Updated the script to print the file content to the War Room.
```

File: Packs/CommonScripts/ReleaseNotes/1_11_32.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+##### ShowIncidentIndicators
+- Fixed an issue where the script would show unrelated indicators on Cortex XSOAR 8.0+.
```
Comment (ShirleyDenkberg) [inline Packs/CommonScripts/ReleaseNotes/1_11_32.md:8]:
```suggestion
Fixed an issue where the script would show unrelated indicators in Cortex XSOAR 8.0+.
```

**Comment by ShirleyDenkberg** [comment]
@dorschw @michalgold Doc review completed.

**Pull request merged**